cvat-ui/src/utils/opencv-wrapper/opencv-wrapper.ts

```diff
@@ -139,6 +139,20 @@ export class OpenCVWrapper {
             hist: () => new HistogramEqualizationImplementation(this.cv),
         };
     }
+
+    public get tracking(): Tracking {
+        if (!this.initialized) {
+            throw new Error('Need to initialize OpenCV first');
+        }
+        return {
+            trackerMIL: {
+                model: () => new TrackerMImplementation(this.cv),
+                name: 'TrackerMIL',
+                description: 'TrackerMIL',
```

Comment:
Maybe provide more detailed description? (Something like: Light client-side model useful to track simple objects)

Reply:
Done